.travis.yml

```diff
@@ -21,7 +21,6 @@ script:
 deploy:
   provider: cargo
   on:
-    repo: CanalTP/transit_model
-    branch: master
+    tags: true
```

Comment:
Is that the kind of change that will now prevent the `master` branch to fail most of the build on TravisCI? :heart_eyes:

Reply:
surely